recon_service_node:
    gridm: 1.2
    holemaxsize: 50
    output_folder: $(find recon_surface)/tmp_reconstructions/
    sample: 1.0
    texturesize: 320.0
    trim: 8.0
    